packages/xsnap/api.js

```diff
@@ -7,7 +7,7 @@
  * Also, update golden master test/test-xs-perf.js to reflect new meter
  * version.
  */
-export const METER_TYPE = 'xs-meter-25';
+export const METER_TYPE = 'xs-meter-26';
```

Comment:
This seems appropriate, and conservative, but maybe not necessary, given that no C code is changing.

Please keep it in, but it does raise a slight question of when exactly we need to bump this, other than obviously when we change the XS (`moddable`) or xsnap (`xsnap-native`) git-submodule hashes. I think the answer is "never", but we do need to remain cognizant of how changes to the supervisor (or liveslots) affects metering of the vat as a whole (we already don't expect two different executions of the same delivery to have the same metering if they report different `METER_TYPE` versions, but we should also not expect them to match if we created the worker with different versions of the supervisor or liveslots, because code inside both is frequently executed as part of the delivery).